components/alert/Alert.tsx

```diff
@@ -18,7 +18,7 @@ export interface AlertProps {
   /** Type of Alert styles, options:`success`, `info`, `warning`, `error` */
   type?: 'success' | 'info' | 'warning' | 'error';
   /** Whether Alert can be closed */
-  closable?: boolean;
+  closable?: boolean | ({ icon?: React.ReactNode } & React.AriaAttributes);
```

Comment:
```suggestion
  closable?: boolean | ({ closeIcon?: React.ReactNode } & React.AriaAttributes);
```

这样是不是好点？https://github.com/ant-design/ant-design/discussions/42828#discussioncomment-6089019

Reply:
确实是，我调整一下